src/libasr/asr_utils.h

```diff
@@ -2100,9 +2443,18 @@ class ReplaceReturnWithGotoVisitor: public ASR::BaseStmtReplacer<ReplaceReturnWi
 
 };
 
+static inline bool present(Vec<ASR::symbol_t*> &v, const ASR::symbol_t* name) {
+    for (auto &a : v) {
+        if (a == name) {
+            return true;
+        }
+    }
+    return false;
+}
+
 // Singleton LabelGenerator so that it generates
 // unique labels for different statements, from
-// wherever it is called (be it ASR passes, be it
+// whereever it is called (be it ASR passes, be it
```

Comment:
I think the previous spelling was correct. (https://prowritingaid.com/grammar/1000219/Whereever-vs-wherever%E2%80%94what-is-the-difference)